srcpkgs/deadbeef/template

```diff
@@ -1,53 +1,65 @@
- libzip-devel mpg123-devel opusfile-devel pulseaudio-devel wavpack-devel"
+ libmad-devel libsamplerate-devel libsndfile-devel libvorbis-devel
+ libzip-devel mpg123-devel opusfile-devel pulseaudio-devel wavpack-devel
+ pipewire-devel libdispatch-devel"
 depends="desktop-file-utils hicolor-icon-theme"
 short_desc="Ultimate Music Player for GNU/Linux"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="Zlib, GPL-2.0-or-later, LGPL-2.1-or-later"
-_apbuild_commit=c64f490e54447e3902fae54784d01ab59fa7dc95
-_mp4p_commit=77ac9f2ecaa3a2f6e232fb45cd7f8550e13cc3d3
+_libretro_commit=b4d3db19566398603069d02eeacb3b06987a1b74
+_output_pw_commit=0b099d13ab0e89d9934aabdeb2872f0f66ea6960
+_mp4p_commit=156195ccb635f016dc34b89425bfbecf046c90d4
 homepage="https://deadbeef.sourceforge.io"
 changelog="https://deadbeef.sourceforge.io/news0.html"
 distfiles="https://github.com/DeaDBeeF-Player/deadbeef/archive/${version}.tar.gz>${pkgname}-${version}.tar.gz
- https://github.com/DeaDBeeF-Player/apbuild/archive/${_apbuild_commit}.tar.gz>apbuild-${_apbuild_commit}.tar.gz
+ https://github.com/DeaDBeeF-Player/ddb_dsp_libretro/archive/${_libretro_commit}.tar.gz>libretro-${_libretro_commit}.tar.gz
+ https://github.com/DeaDBeeF-Player/ddb_output_pw/archive/${_output_pw_commit}.tar.gz>output_pw-${_output_pw_commit}.tar.gz
  https://github.com/DeaDBeeF-Player/mp4p/archive/${_mp4p_commit}.tar.gz>mp4p-${_mp4p_commit}.tar.gz"
-checksum="a667d55b787d76bc9d3593882b57ae9a449a99e93e1c11f85ec638566e10877d
- c5eeb3fa116d51d251fe7c7e5c44798680f6478c43a1fffdcb28c4caf1c68286
- d0ba1b4ad062cb5c056c4a59ed55769d79371fc9b3bf9aaa0ca48f508e3eba75"
-LDFLAGS+=" -Wl,-z,stack-size=1048576"
+checksum="53ed535335e637437adf77eb4bcadb781eddbf5539f74373d5a863389df5e2d0
+ 444d4d89edbd51b9d2305c83a49e18949e0f21a42eec2a95ce03efd752a81049
+ 59115ddcd0378aa2f5914138c5c256198d66339bfbb3d65389b9bf4fa327f9ee
+ 3b5bdbcb2808d12b9f5af630a91e77be1036aeb487d5fa0a323ce8080918439b"
+skip_extraction="libretro-${_libretro_commit}.tar.gz
+ output_pw-${_output_pw_commit}.tar.gz
+ mp4p-${_mp4p_commit}.tar.gz"
 
 # The soundtouchup plugin was made default in 1.8.4 and fails to build on i686.
 # Fixing that might be a good maintainer task.
 case "$XBPS_TARGET_MACHINE" in
-	i686*) configure_args+=" --disable-soundtouch" ;;
+	i686*) configure_args+=" --disable-soundtouch --disable-ffap" ;;
 esac
 
 if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
 	makedepends+=" musl-legacy-compat"
 fi
 
 post_extract() {
-	mv deadbeef-${version}/* .
-	rm -rf external/apbuild
+#	mv deadbeef-${version}/* .
 	rm -rf external/mp4p
-	mv -v apbuild-${_apbuild_commit} external/apbuild
-	mv -v mp4p-${_mp4p_commit} external/mp4p
+	rm -rf external/ddb_dsp_libretro
+	rm -rf external/ddb_output_pw
+	vsrcextract -C external/ddb_dsp_libretro libretro-${_libretro_commit}.tar.gz
+	vsrcextract -C external/ddb_output_pw output_pw-${_output_pw_commit}.tar.gz
+	vsrcextract -C external/mp4p p4p-${_mp4p_commit}.tar.gz
```

Comment:
```suggestion
	vsrcextract -C external/mp4p mp4p-${_mp4p_commit}.tar.gz
```

Reply:
that **m** ah. k think most of these are covered